### PR TITLE
Properly escape property value to handle non string types

### PR DESCRIPTION
**Pull request by stephenmichaelf**

### CODE REVIEW

**Comment by ericsciple** [comment]
Looks good can you add the same comment from below in the file and an L0

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by stephenmichaelf** [comment]
@ericsciple Comment and tests added.

File: node/taskcommand.ts [hunk 1]
```diff
@@ -32,7 +32,9 @@ export class TaskCommand {
                 if (this.properties.hasOwnProperty(key)) {
                     var val = this.properties[key];
                     if (val) {
-                        cmdStr += key + '=' + escape(val) + ';';
+                        // safely append the message - avoid blowing up when attempting to
```
Comment (ericsciple) [inline node/taskcommand.ts]: `message` => `val`

**Comment by ericsciple** [comment]
looks good

**Comment by bryanmacfarlane** [comment]
published as 2.2.1

**Pull request merged**